### DISCUSSION TITLE
Prerelease 0.9.2 testing

### DISCUSSION THREAD

**Discussion by benni092** [rg35xx-cfw/Koriki#68] [General]
Hello to all koriki fans out there!

I have done some quick tests with the 0.9.2 system folder:

In PPSSPP-sa the analog stick is not mapped
(tested this with Daxter, you go in first person mode and cant move)

In mupen64plus-sa only the c-up is mapped. All other c-buttons are not mapped.
(tested this with Zelda OOT, you cant play the ocarina) And audio is very choppy. (on my normal system folder i turned off v-sync and audio-sync in mupen64plus-sa)
If someone wants to try my configs i upload it.

[configs.zip](https://github.com/rg35xx-cfw/Koriki/files/12851102/configs.zip)

bw

**Comment by acmeplus** [comment]
Thanks for the feedback, I'll have a look at those config and update them as needed.

**Reply by paladinperry** [reply]
Sorry to jump in again but I solely use koriki it's by far the best looking firmware in 35xx.

Do you have any idea when 1.0 will be ready for release and what to expect ?

**Reply by acmeplus** [reply]
Pretty soon(TM)
